src/goto-symex/goto_symex.cpp

```diff
@@ -27,6 +30,21 @@ void goto_symext::replace_nondet(exprt &expr)
   if(expr.id()==ID_side_effect &&
      expr.get(ID_statement)==ID_nondet)
   {
+    null_message_handlert null_message;
```

Comment:
Can we route a handler in instead of throwing this away?

Reply:
Will be solved by #1625, for the moment I'll keep an extra patch that keeps this `null_message_handlert`.